packages/api-cardano-db-hasura/src/typeAliases.ts

```diff
@@ -1,3 +1,19 @@
+import { ComplexityEstimator } from 'graphql-query-complexity'
```

Comment:
Best to put this into a separate file named `queryComplexity`. `typeAliases` is just meant for conveniences such as more specific types the omit or pick fields.